### PR TITLE
add local attractions to san fran file

### PR DESCRIPTION
**Pull request by madrone1**
this is a pull request

### CODE REVIEW

File: madrone1-san_francisco.md [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# San Francisco
+
+### Restaurants
```
Comment (shinton) [inline madrone1-san_francisco.md]: What's your favorite restaurant?

File: madrone1-san_francisco.md [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+
+### Local Attractions
+
+- Delores park
+- AT&T park
+
```
Comment (yashkmankad) [inline madrone1-san_francisco.md]: Golden Gate Bridge
Fisherman's Wharf
Pier 39
Lombard Street

Comment (GolfDiva) [inline madrone1-san_francisco.md]: BART!

**Comment by SmilyBorg** [comment]
Isn't Folsum Street there?

**Pull request merged**